File: responses/Z9S2T8-8XQ/reverse_digits.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(X) {
    var isPositive = X >= 0
    var xString = Math.abs(X).toString()
    
    var split = xString.split("")
    
    var reversed = split.reverse()
    
    var parsed = parseInt(reversed.join(""))
    
    if(isPositive)
        return parsed
    else
        return parsed * -1
}

File: responses/HH4HT2-UZB/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X):
    new_text = ''
    pos = -1
    X = str(X)
    while pos * -1 <= len(X):
        if X[pos] != '-':
            new_text = new_text + X[pos]
        else:
            new_text = '-' + new_text
        pos -=1
    return int(new_text)